tower-balance/src/load/peak_ewma.rs

```diff
@@ -80,11 +80,7 @@ const NANOS_PER_MILLI: f64 = 1_000_000.0;
 
 // ===== impl PeakEwma =====
 
-impl<D, I> WithPeakEwma<D, I>
-where
-    D: Discover,
-    I: Instrument<Handle, D::Response>,
```

Comment:
The bounds on this impl block had to be removed because the `Discover` trait was made generic, and the type parameter `R` for `Discover`'s request type would be unbounded w/o `PhantomData`.